FieldOpt/Hdf5SummaryReader/hdf5_summary_reader.cpp

```diff
@@ -8,27 +8,176 @@ Hdf5SummaryReader::Hdf5SummaryReader(const std::string file_path)
   DATASET_NAME_TIMES("TIMES"),
   GROUP_NAME_FLOW_TRANSPORT("FLOW_TRANSPORT"),
   DATASET_NAME_ACTIVE_CELLS("ACTIVE_CELLS"),
-  DATASET_NAME_WELL_STATES("WELL_STATES")
+  DATASET_NAME_WELL_STATES("WELL_STATES"),
+  DATASET_NAME_PRESSURE("PTZ"),
+  DATASET_NAME_SATURATION("GRIDPROPTIME")
 {
     readTimeVector(file_path);
-    readActiveCells(file_path);
     readWellStates(file_path);
+    readActiveCells(file_path);
```

Comment:
I'll add a flag so that this is only activated when compiling the standalone H5-ConversionTool; otherwise it is off, unless we call it explicitly at end of optimization run (to do later).